src/libraries/System.IO.Ports/tests/SerialPort/Read_char_int_int.cs

```diff
@@ -34,6 +34,9 @@ public class Read_char_int_int : PortsTest
         //The maximum buffer size when an exception is not expected
         private const int maxBufferSize = 8;
 
+        // The web name of Encoding.UTF7
+        private const string utf7WebName = "utf-7";
```

Comment:
Utf7WebName

Reply:
I'm following the convention already present in this file.

Reply:
I would say for this one, we don't have to be consistent with the rest of the file and Use the Camel case better.

---
In reply to: [436877176](https://github.com/dotnet/runtime/pull/37535#discussion_r436877176) [](ancestors = 436877176)

Reply:
This field no longer exists as of newest iteration.